### DISCUSSION TITLE
Doesn't work in Firefox. PR_CONNECT_RESET_ERROR

### DISCUSSION THREAD

**Discussion by JKDos** [jlai/fitness-dashboard#30] [General]
Still to this day the page refuses to load in Firefox. 

Seeing

```
Secure Connection Failed

An error occurred during a connection to dashboard.exercise.quest. PR_CONNECT_RESET_ERROR

Error code: PR_CONNECT_RESET_ERROR

    The page you are trying to view cannot be shown because the authenticity of the received data could not be verified.
    Please contact the website owners to inform them of this problem.
```

![image](https://github.com/user-attachments/assets/504ae9f8-c4a0-4a7d-b769-ba5608637d8f)

**Comment by gramapp** [comment]
Hm, works for me on firefox 123.0.1.

**Reply by JKDos** [reply]
I'm on the newest version. 128.0.2 (64-bit).

**Comment by jlai** [comment]
Do you have any ad blocker extensions or VPNs? We don't have ads but it's possible some bad extension is messing with the connection.

**Reply by JKDos** [reply]
No adblockers in Firefox, only these. I do use ad blockers in Chrome, and Chrome has no problem loading the page. I disabled the 3 extensions in Firefox, but it did not solve the problem.

![image](https://github.com/user-attachments/assets/6b1bbb2d-168f-4dce-aff2-042b1471628c)

ETA: no VPNS.

**Reply by jlai** [reply]
That's really weird then. Only other thing I can think of is turning on DNS-over-HTTP if it isn't enabled, in case your ISP is messing with it or some DNS cache it out of date.

**Reply by chrissard2** [reply]
I just dwnloaded firefox and have no problems
![image](https://github.com/user-attachments/assets/f818231c-4fc1-4193-b5c9-a8ff5e463094)

**Comment by chrissard2** [comment]
Have you tried to reset Firefox?

Reset Firefox
This option resets Firefox to factory defaults, removing customizations like extensions and themes:
Click the orange Firefox button in the top left corner
Hover over Help
Click Troubleshooting Information
Click Reset Firefox in the top right of the page
Click Reset Firefox to confirm
Click Finish when finished
